### DISCUSSION TITLE
Thanks for the head start :)

### DISCUSSION THREAD

**Discussion by joekm** [JaKooLit/Fedora-Hyprland#59] [General]
Started with an older set of your scripts.  They helped me understand how Hyprland works well enough to give me a good start in learning how to make my own setup.  These days, I'm pretty much running my own stuff and definitely still have a lot to learn but, that's kinda the appeal of Hyprland.  Gives me something to "tinker with" :)

**Comment by JaKooLit** [comment]
cool. Yeah that is actually my aim. To help startup... and create each own config to satisfy each needs

**Comment by gwicho38** [comment]
Using this thread for the same purpose. Thanks :+1: ! With two configs, trudging on my own configs is a slog. Cheers :)

**Reply by JaKooLit** [reply]
> Using this thread for the same purpose. Thanks 👍 ! With two configs, trudging on my own configs is a slog. Cheers :)

Thanks for sponsoring :)
